### DISCUSSION TITLE
Does SystemAccentColor only work for WinUI/UWP platform?

### DISCUSSION THREAD

**Discussion by nathanAjacobs** [unoplatform/uno#8949] [Q&A] [answered]
The SystemAccentColor shows as blue in WPF Skia app and not the system accent color set in Windows Settings. I'm guessing this is only possible for WinUI / UWP targets.

**Comment by jeromelaban** [comment, marked as answer]
This feature is not yet implemented: https://github.com/unoplatform/uno/issues/4444
